### PR TITLE
chore: enable `verbatim-module-syntax` lint rule

### PR DESCRIPTION
**Pull request by dsherret**

### CODE REVIEW

File: front_matter/mod.ts [hunk 1]
```diff
@@ -115,9 +115,9 @@
  *
  * @module
  */
-import { extract as extractJson } from "./json.ts";
-import { extract as extractToml } from "./toml.ts";
-import { extract as extractYaml } from "./yaml.ts";
+import type { extract as extractJson } from "./json.ts";
+import type { extract as extractToml } from "./toml.ts";
+import type { extract as extractYaml } from "./yaml.ts";
```
Comment (dsherret) [inline front_matter/mod.ts:120]: Huh, interesting.

Comment (iuioiua) [reply front_matter/mod.ts:120]: Omg. We import these symbols without exporting them. Lol. If these were exported, it'd break their functionality. Let's remove these imports for now. I'll fix this up in a separate PR.

Comment (dsherret) [reply front_matter/mod.ts:120]: I think this is done for the module doc above? Not sure.

Comment (iuioiua) [reply front_matter/mod.ts:120]: Oh... Yes. That might be right.

**Review by iuioiua — APPROVED** [review]
LGTM

**Pull request merged**